activerecord/test/cases/query_logs_formatter_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+
+class QueryLogsFormatter < ActiveRecord::TestCase
+  def test_factory_invalid_formatter
+    assert_raises(ArgumentError) do
+      ActiveRecord::QueryLogs::FormatterFactory.from_symbol(:non_existing_formatter)
+    end
+    end
+
+  def test_factory_invalid_quote_values
+    assert_raises(ArgumentError) do
+      ActiveRecord::QueryLogs::Formatter.new(key_value_separator: ":", quote_values: :does_not_exist)
```

Comment:
If we don't expect anyone to directly instantiate the class, I think we should remove this test.